app/models/remote_theme.rb

```diff
@@ -214,6 +205,11 @@ def github_repo_url
       "https://github.com/#{org_repo}"
     end
   end
+
+  def is_git
```

Comment:
Minor but in ruby we prefer boolean methods to end with `?`. Also the body could be `remote_url.present?`